### PR TITLE
Update doc for edd_get_users_purchases() and allow array to be passed as $status

### PR DESCRIPTION
**Pull request by sunnyratilal**
PR for #5163

### CODE REVIEW

File: includes/user-functions.php [hunk 1]
```diff
@@ -38,7 +37,14 @@ function edd_get_users_purchases( $user = 0, $number = 20, $pagination = false,
 		return false;
 	}
 
-	$status = $status === 'complete' ? 'publish' : $status;
+	if ( ! is_array( $status ) && is_string( $status ) ) {
```
Comment (chriscct7) [inline includes/user-functions.php]: Couldn't you just do is_string here?

Comment (cklosowski) [inline includes/user-functions.php]: Yeah the `! is_array` isn't needed, should we support a comma separate list of statuses here too? So something like:
```
if ( is_string( $status ) ) {
    if ( strpos( ',', $status ) ) {
        $status = explode( ',', $status );
    } else {
        $status = $status === 'complete' ? 'publish' : $status;
        $status = array( $status );
    }
}

Comment (cklosowski) [inline includes/user-functions.php]: Only reason I say this is that many functions in WP Core allow an array, or comma separated list of statuses. Just makes it consistant and doesn't take much on our end.

**Comment by cklosowski** [comment]
@chriscct7 Look better? @sunnyratilal if you are good with this merge when you review.

**Comment by sunnyratilal** [comment]
All good here

**Pull request merged**